napari/layers/base/_base_constants.py

```diff
@@ -116,3 +116,13 @@ def corners(cls):
             cls.BOTTOM_LEFT,
             cls.BOTTOM_RIGHT,
         )
+
+
+class ActionType(StringEnum):
+    """
+    Action types for layer.events.data of Shapes and Points layer.
+    """
+
+    ADD = auto()
+    REMOVE = auto()
+    CHANGE = auto()
```

Comment:
please reexport this in `napari/layers/base/__init__.py` to make it part of public interface and allow people to use it in code.

Reply:
resolved with [5173255](https://github.com/napari/napari/pull/5967/commits/51732552282c6873c634b3a6cd39bb02dc769019)